samza-api/src/main/java/org/apache/samza/system/ExtendedSystemAdmin.java

```diff
@@ -26,7 +26,7 @@
  * TODO: Merge this interface method with SystemAdmin when we upgrade to JDK 1.8
  */
 public interface ExtendedSystemAdmin extends SystemAdmin {
-  Map<String, SystemStreamMetadata> getSystemStreamPartitionCounts(Set<String> streamNames, long cacheTTL);
+  Map<String, Integer> getSystemStreamPartitionCounts(Set<String> streamNames, long cacheTTL);
```

Comment:
Are we planning to use this in 0.14 or a minor version? This is a breaking change in API and if we are breaking it, I would rather remove this interface and merge this method in the standard SystemAdmin interface.